.eslintrc

```diff
@@ -0,0 +1,213 @@
+ecmaFeatures:
```

Comment:
One more comment: you may want to rename this file into **.eslintrc.yaml** or **.eslintrc.yml**. According to:
http://eslint.org/docs/user-guide/configuring#configuration-file-formats
The name **.eslintrc** is deprecated. (I think it is a good idea, because without the extra extension, we have to guess what syntax this file is written in.)